server/routes/api/boycottList.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-const BoycottList = require("../../../db/models/boycottList");
+const BoycottList = require("../../../db/models/boycottLists");
 
 router.get("/test", (req, res) => {
```

Comment:
I think we're about ready to add the endpoint for finding a boycotted business by name, so you can probably add that here as well

Reply:
Yeah. I believe I did that already but maybe It's after this PR. I'll double-check to make sure it is in the next PR